### ISSUE TITLE
[Errata]: Incorrect description of Pin layout

### ISSUE THREAD

**Issue by Rueful-Pi** [raspberrypipress/official-raspberry-pi-beginners-guide-5e#9]
### What format of the book has this error? You may select more than one option.

Print

### What language of the book has this error?

English (EN)

### Page Number or Section

Chapter 9 page 228

### Detailed description

The text explains that, "Pin 19 is the bottom-left, Pin 20 the bottom-right". While the Figure 9-12 shows pins 1-20 on left side and pins 21-40 on the right. This same mistake is also made in 'Getting started with MicroPhython on Raspberry Pi Pico" by Garath Halfacree and Ben Everard (RPi_PiPico_Digital_v11.pdf), on page 35.

**Comment by Rueful-Pi** [comment]
Will there be new projects in the 2nd Edition?

**Comment by bjepson** [comment]
Yes! We're replacing the LCD display with more affordable, and more widely available SSD1306 OLEDs, and have also added a couple of Wi-Fi projects and some BLE example code.

**Comment by Rueful-Pi** [comment]
Sounds awesome, I'm looking forward to that. Wi-Fi projects and BLE examples will be excellent!

**Comment by bjepson** [comment]
A belated update/closed issue; we corrected this in the 2nd printing of this book (July 2024).
![Screenshot 2025-01-06 at 10 47 13 am](https://github.com/user-attachments/assets/9b57c147-f218-4542-80fe-44df662e27d0)
